src/mewbot/tools/__init__.py

```diff
@@ -98,8 +105,10 @@ def __call__(self) -> None:
 
         issues = list(self.run())
 
-        if self.is_ci:
-            self.github_list(issues)
+        with open(
```

Comment:
It was previously only in CI to stop the annotations being output locally, as they're not exactly readable. But as they now go to file, the need for different behaviour is diminished.

After these changes, the differences enabling the CI flag makes are:
 - `black` outputs a diff rather than auto fixing.
 - `mypy` has colour and pretty output disabled
 - `pytest` does not generate HTML coverage